torchgeo/datasets/mapinwild.py

```diff
@@ -0,0 +1,479 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""MapInWild dataset."""
+
+import os
+import shutil
+from collections.abc import Sequence
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import rasterio
+import torch
+from einops import rearrange
+from torch import Tensor
+
+from torchgeo.datasets.geo import NonGeoDataset
+from torchgeo.datasets.utils import (
+    check_integrity,
+    download_url,
+    extract_archive,
+    percentile_normalization,
+)
+
+
+class MapInWild(NonGeoDataset):
+    """MapInWild dataset.
+
+    The `MapInWild <https://arxiv.org/abs/2212.02265>`_ dataset is curated for the task
```

Comment:
```suggestion
    The `MapInWild <https://arxiv.org/abs/2212.02265>`__ dataset is curated for the task
```
Otherwise the link in the https://torchgeo--1131.org.readthedocs.build/en/1131/api/datasets.html#non-geospatial-datasets table will take you to the paper instead of the API docs.